### PR TITLE
[full-ci] Fix ci waiting times

### PR DESCRIPTION
**Pull request by kulmann**
CI is waiting a lot at the moment. This PR tries to reduce that wherever possible. Mainly it fixes an unneeded 10 seconds waiting in the right sidebar for a `back` button when already being in the root panel.

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/18025/11/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by paulcod3 — APPROVED** [review]
LGTM

**Comment by ownclouders** [comment]
Results for <strong>oC10MoveUpload</strong> https://drone.owncloud.com/owncloud/web/18032/14/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:152

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/18032/52/1
The following scenarios passed on retry:
- webUIMarkdownEditor/markdownFile.feature:19

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -553,7 +553,7 @@ When('the user batch declines these shares using the webUI', async function(file
     await client.page.FilesPageElement.filesList().toggleFileOrFolderCheckbox('enable', item[0])
   }
 
-  return client.page.personalPage().declineAllCheckedShares()
+  return client.page.sharedWithMePage().batchDeclineShares()
```
Comment (phil-davis) [inline tests/acceptance/stepDefinitions/filesContext.js:556]: Well found. This looks like it was an existing error, not related to the other timing problems.

**Review by phil-davis — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>oC10iPhone2</strong> https://drone.owncloud.com/owncloud/web/18058/46/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:90

File: tests/acceptance/stepDefinitions/filesContext.js (hunk 1 above)
Comment (kulmann) [reply tests/acceptance/stepDefinitions/filesContext.js:556]: Yes, that was weird. Was either hard failing, or had an error message in the console while succeeding. That was an unrelated flaky test and I took the chance to solve it. 😎

Comment (kulmann) [reply tests/acceptance/stepDefinitions/filesContext.js:556]: (If I remember correctly I introduced it myself 😁 )

**Review by pascalwengerter — APPROVED** [review]
Good to go if CI turns green

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait2</strong> https://drone.owncloud.com/owncloud/web/18071/43/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:90
- webUIUpload/upload.feature:99

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait2</strong> https://drone.owncloud.com/owncloud/web/18075/43/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:90

**Comment by ownclouders** [comment]
Results for <strong>oC10iPhone2</strong> https://drone.owncloud.com/owncloud/web/18075/46/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:99

**Comment by ownclouders** [comment]
Results for <strong>oC10MoveUpload</strong> https://drone.owncloud.com/owncloud/web/18077/14/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:90

**Comment by kulmann** [comment]
CI is green now. Thanks @fschade for taking care of the upload tests! ❤️

**Pull request merged**